gamemodes/cinema/gamemode/modules/theater/services/sh_hls.lua

```diff
@@ -70,15 +72,15 @@ if CLIENT then
 	
 	function SERVICE:LoadVideo( Video, panel )
 		local k = Video:Key()
-		if (string.len(Video:Data())>1 and Video:Data() != "true") then
+		if (not isbool(Video:Data()) and string.len(Video:Data())>1) then
```

Comment:
Can we please keep the Data type as a string to prevent potentially breaking other code?